assert/assertions.go

```diff
@@ -22,7 +22,7 @@ import (
 	yaml "gopkg.in/yaml.v2"
 )
 
-//go:generate go run ../_codegen/main.go -output-package=assert -template=assertion_format.go.tmpl
+//go:generate sh -c "cd ../_codegen && go build && cd - && ../_codegen/_codegen -output-package=assert -template=assertion_format.go.tmpl"
```

Comment:
If we're rerunning this command several times in a few files, would it make more sense to separate the commands into a bash file and have the `go:generate` tool call the same file but with different parameters?

I think that would make the code cleaner & easier to maintain as well as reused with very little added extra complexity.

Reply:
I think this line is low frequency enough that the value in doing that isn't high. It would mean another script floating around somewhere in the repo. I don't feel strongly though. Happy to add it if the maintainers would like it. @glesica?

Reply:
Fair point. I'd be happy with it either way as well then. I guess if it proves problematic we can look into separating it out. 👍

Reply:
Sounds good to me.